design-system/tokens/src/utils/semantic.mjs

```diff
@@ -4,7 +4,7 @@ function inputColor(name) {
   return {
     color: createColor(`{intents.base.12}`),
     placeholder: createColor(`{intents.base.10}`),
-    border: createColor(`{intents.${name}.8}`),
+    border: createColor(`{intents.${name}.6}`),
```

Comment:
This things will also be reflected on the figma?

Reply:
of course